sympy/solvers/ode.py

```diff
@@ -8122,15 +8122,15 @@ def _nonlinear_3eq_order1_type2(x, y, z, t, eq):
     y_z = sqrt(((a*C1-C2) - c*(a-c)*z(t)**2)/(b*(a-b)))
     try:
         sol1 = dsolve(a*diff(x(t),t) - (b-c)*y_x*z_x*r[f]).rhs
-    except:
+    except Exception:
```

Comment:
You should specific in the type of `Exception` you want to raise. So is the case below.